go/vt/sqlparser/sql.y

```diff
@@ -4451,6 +4451,9 @@ assignment_expression:
   {
```

Comment:
do all the tests still pass if you remote the column name, and only have `reserved_keyword = ...`?